input/data/2011/2011A%26A...531L..18H/tev-000109.yaml

```diff
@@ -0,0 +1,25 @@
+source_id: 83
+reference_id: 2011A&A...531L..18H
+
+data:
+  livetime: 62
+  significance: 7.5
+
+pos:
+  ra: {val: 17h47m49s, err: 1.8m, err_sys: 1.3s}
+  dec: {val: -24d48m30s, err: 36s, err_sys: 20s}
+
+morph:
+  type: gauss
+
+  sigma: {val: 9.6, err: 2.4}
+  sigma2: {val: 1.8, err: 1.2}
+  # TODO: frame type {galactic, radec} not found in the paper
```

Comment:
The frame is Galactic.
TODO comment should stay, but mention that it's "westwards from north" in the paper, and that we still have to decide which orientation to use in gamma-cat data entry.